diff -up sip-4.19.10/configure.py.omv~ sip-4.19.10/configure.py
--- sip-4.19.10/configure.py.omv~	2018-06-25 17:22:42.421920036 +0200
+++ sip-4.19.10/configure.py	2018-06-25 17:23:00.320920703 +0200
@@ -320,7 +320,7 @@ def create_makefiles(macros):
     distinfo_dir = os.path.join(cfg.sip_root_dir,
             '%s-%s.dist-info' % (sip_module_name.replace('.', '_'),
                     sip_version_str))
-    mk_distinfo = '%s mk_distinfo.py %s installed.txt' % (sys.executable,
+    mk_distinfo = '%s mk_distinfo.py $(DESTDIR)%s installed.txt' % (sys.executable,
             distinfo_dir)
 
     if opts.use_qmake:
--- sip-4.19.10/mk_distinfo.py	2018-06-23 11:44:14.000000000 +0200
+++ sip-4.19.10/mk_distinfo.py	2018-06-25 17:35:10.144947918 +0200
@@ -98,7 +98,10 @@
         else:
             fn_name = fn
 
-        fn_f = open(fn, 'rb')
+        try:
+            fn_f = open(os.getenv("DESTDIR") + fn, 'rb')
+        except:
+            fn_f = open(fn, 'rb')
         data = fn_f.read()
         fn_f.close()
 
